### PR TITLE
ROC-5439 using an expert in small claims

### PR DESCRIPTION
**Pull request by sudheer-ajax**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5439

### Change description

Added expert guidance static page

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/directions-questionnaire/views/expert-guidance.njk [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "form.njk" import csrfProtection, submitButton, errorSummary %}
+
```
Comment (robertparkinson) [inline src/main/features/directions-questionnaire/views/expert-guidance.njk]: Please remove one of these empty lines

File: src/main/features/directions-questionnaire/views/expert-guidance.njk [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
```
Comment (robertparkinson) [inline src/main/features/directions-questionnaire/views/expert-guidance.njk]: supportWidget doesn't seem to be used on this page?

File: src/test/features/directions-questionnaire/routes/expert-guidance.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import { Paths } from 'directions-questionnaire/paths'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import { checkAuthorizationGuards } from 'test/features/ccj/routes/checks/authorization-check'
+import { RoutablePath } from 'shared/router/routablePath'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
```
Comment (andrewwa-kainos) [inline src/test/features/directions-questionnaire/routes/expert-guidance.ts]: Please refactor relative imports

File: src/test/features/directions-questionnaire/routes/expert-reports.ts [hunk 4]
```diff
@@ -154,8 +155,8 @@ describe('Directions Questionnaire - expert reports page', () => {
           await request(app)
             .post(pagePath)
             .set('Cookie', `${cookieName}=ABC`)
-            .send(validDeclaredFormData)
-            .expect(res => expect(res).to.be.redirect.toLocation(selfWitnessPage))
+            .send(validDeclinedFormData)
+            .expect(res => expect(res).to.be.redirect.toLocation(expertGuidance))
```
Comment (andrewwa-kainos) [inline src/test/features/directions-questionnaire/routes/expert-reports.ts]: The name of this test should be updated accordingly

**Pull request merged**